html_reference.html

```diff
@@ -1,9 +1,10 @@
 <!DOCTYPE html>
-<html>
+<html lang='en'>
 	<head>
 		<link rel="stylesheet" href="css_stylesheet.css">
 		<script src="https://kit.fontawesome.com/c90803ff49.js" crossorigin="anonymous"></script>
 		<title>milo_girardet</title>
```

Comment:
This is a badly-formatted title. Remember that the title is visible to the user as the wepage's name in the browser tab. You'll want it to say something like Milo Girardet

Reply:
On the page itself, it's obvious that this is part of a conscious formatting choice, but it will be less obvious if the user has multiple tabs open. This one's not particularly high priority if you prefer it the way it is right now.

Reply:
Good point, have now changed.